src/routes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+$config = array_merge(config('translation-manager.route'), ['namespace' => 'Barryvdh\TranslationManager']);
```

Comment:
I made it work through this.

It looks a bit ugly since we are not configuring in ServiceProvider anymore